### PR TITLE
feat: increase tooltip width to 500px and truncate items to 2 lines

### PR DESCRIPTION
**Pull request by mbondyra**
## Summary

Fixes https://github.com/elastic/elastic-charts/issues/2048 (I am not sure if we consider this issue fixed after merging this one)

Auto-expands the tooltip containter to 500px for wider content. The data items in the tooltip now truncate to 2 lines instead of just one. 

Screenshot 2023-11-28 at 15 06 15

Screenshot 2023-11-28 at 15 05 32

### CODE REVIEW

File: packages/charts/src/components/tooltip/components/_tooltip.scss [hunk 1]
```diff
@@ -56,7 +56,11 @@ $transitionTime: 200ms;
   }
 
   .echTooltip__tableCell--truncate {
-    @include euiTextTruncate;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    line-clamp: 2;
+    -webkit-line-clamp: 2;
+    overflow: hidden;
```
Comment (markov00) [inline packages/charts/src/components/tooltip/components/_tooltip.scss:63]: you can probably use also the `lineClamp` sass mixin, it has a few more properties but it should work

File: packages/charts/src/chart_types/metric/renderer/dom/_text.scss [hunk 2]
```diff
@@ -1,12 +1,3 @@
-@mixin lineClamp($maxLines) {
```
Comment (mbondyra) [inline packages/charts/src/chart_types/metric/renderer/dom/_text.scss:1]: this was not used in any place so I moved it to generic mixins

**Review by markov00 — CHANGES_REQUESTED** [review]

**Comment by mbondyra** [comment]
Thank you @markov00 great tips! I addressed all your comments! 🙇🏼‍♀️

**Review by markov00 — APPROVED** [review]
Changes looks good to me, tested on deployed storybook.
Just re-generate the changed screenshots from the CI rather than from the local machine to correctly match with what the CI is going to match.

**Comment by mbondyra** [comment]
buildkite update vrt

**Pull request merged**